src/main/java/ro/msg/learning/shop/CSV2JSON/WebConfiguration.java

```diff
@@ -0,0 +1,17 @@
+package ro.msg.learning.shop.CSV2JSON;
+
+        import org.springframework.context.annotation.Configuration;
+        import org.springframework.http.converter.HttpMessageConverter;
+        import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+        import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+        import java.util.List;
+
+@EnableWebMvc
+@Configuration
+public class WebConfiguration implements WebMvcConfigurer {
```

Comment:
This is configuration class optional. Instead you could simply annotated your `CSVconverter` with `@Component`.